pykube/config.py

```diff
@@ -152,7 +157,11 @@ def user(self):
         """
         if self.current_context is None:
             raise exceptions.PyKubeError("current context not set; call set_current_context")
-        return self.users[self.contexts[self.current_context]["user"]]
```

Comment:
I am not sure what the intention of rewriting this line. It looks as if the code you rewrote it to is identical in behavior.